src/main/java/app/model/agents/AgentImp.java

```diff
@@ -23,6 +23,7 @@ public class AgentImp implements Agent
 {
     @Getter @Setter protected double maxWalk = 10;
     @Getter @Setter protected double maxSprint = 30;
+    @Getter @Setter protected double moveLength = 20;
```

Comment:
This addition of `moveLength` as a variable affects only the ACO agent, but has been put in the AgentImp; why?
If it is not used in part of another agent, then this is not following the principles of encapsulation and is bad practice.  (I do however agree with the name change `distance` -> `moveLength`, far better naming)

Reply:
it also affects WF agent and DirectionFollow agent, they just already had their own moveLength. For ACO-agent I had to rename all the uses before being able to replace it, while for the WF- and DF-agent I could just delete the variable from that class

Reply:
*delete the variable construction, so they use the one from AgentImp instead of their own local ones now